tests/utility_tests.py

```diff
@@ -0,0 +1,149 @@
+import cartopy.crs as ccrs
+from clouddrift.utility import plot_ragged
+import matplotlib.pyplot as plt
```

Comment:
Is there a mechanism to skip these tests if optional dependencies are not installed?

Reply:
True, I assumed that it was installed in the ci/cd, but I see that this wouldn't work if you don't have those packages locally.

Reply:
Can you try again with the latest modification?